lib/coarse_op.cuh

```diff
@@ -72,6 +73,7 @@ namespace quda {
       case COMPUTE_CONVERT:
       case COMPUTE_RESCALE:
       case COMPUTE_CLOVER_INV_MAX: // FIXME
+      case COMPUTE_TWISTED_CLOVER_INV_MAX: // FIXME
```

Comment:
DId this get fixed?

Reply:
This is a minor FIXME, and isn't worth worrying about there I think.  It's just the flops for computing the max aren't being computed: mainly because there is a the question whether we special case for dynamic clover or not.  Regardless, the `bytes` is the important metric here, since this will be a bandwidth bound kernel.